engine/src/flutter/shell/common/shell_unittests.cc

```diff
@@ -4946,6 +4946,8 @@ TEST_F(ShellTest, ProvidesNullEngineId) {
   configuration.SetEntrypoint("providesEngineId");
   RunEngine(shell.get(), std::move(configuration));
 
+  ASSERT_EQ(shell->GetEngine()->GetLastEngineId(), 99);
```

Comment:
99 seems like a magic number here. `RunConfiguration::SetEngineId` to something else and assert that instead?

Also, run it again with a different ID to see that it stick? That way, this test should fail without your patch. As it stands, this test should work with or without your patch right?

Reply:
Ouch, that statement should be in different test. Hence the 99 magical value. The test tests for `LastEngineId()`. There is no `LastEngineId` without the patch, it was introduced in the PR.